readme/extending-blt.md

```diff
@@ -36,10 +36,16 @@ You may disable any BLT command. This will cause the target to be skipped during
       disable-targets:
         validate:
           phpcs: true
-        git:
-          commit-msg: true
 
-This snippet would cause the `tests:phpcs:sniff:all` and `internal:git-hook:execute:commit-msg` targets to be skipped during BLT builds.
+This snippet would cause the `tests:phpcs:sniff:all` target to be skipped during BLT builds.
+
+You may also disable some behaviors by overriding their value to `false`. For example, in order disable the Git commit message validation:
+
+      git:
```

Comment:
This isn't really a general example of how to disable a command. It's specific to disabling git hooks only--nothing else uses this configuration pattern.